tests/www/apply/test_process.py

```diff
@@ -3965,22 +4006,31 @@ def test_reload_qualification_fields_404(client):
     company = CompanyFactory(kind=CompanyKind.GEIQ, with_membership=True)
     employer = company.members.first()
     client.force_login(employer)
-    url = reverse("apply:reload_qualification_fields", kwargs={"company_pk": 0})
-    response = client.post(
-        url,
-        data={
-            "guidance_days": "0",
-            "contract_type": ContractType.APPRENTICESHIP,
-            "contract_type_details": "",
-            "nb_hours_per_week": "",
-            "hiring_start_at": "",
-            "qualification_type": job_applications_enums.QualificationType.CQP,
-            "qualification_level": "",
-            "planned_training_hours": "0",
-            "hiring_end_at": "",
-            "answer": "",
-        },
+    job_seeker = JobSeekerFactory(for_snapshot=True)
```

Comment:
On ne prend pas de snapshot.
```suggestion
    job_seeker = JobSeekerFactory()
```